libraries/commerce/favorites/actions/toggleFavorites.js

```diff
@@ -42,7 +42,7 @@ const addFavorites = productId => (dispatch, getState) => {
  * @param {string} productId Product identifier
  * @returns {Promise} PipelineRequest dispatch.
  */
-const syncAddSingleFavorite = productId => (
+const dispatchAddFavorite = productId => (
```

Comment:
I think there was a misunderstanding with this one.
When an action calls a pipeline directly, it should not be prefixed. `addFavorite ` would be the correct name in this case.
The only place where the dispatch prefix was supposed to be used is an action which itself dispatches another action and is not directly responsible for data being changed.

Reply:
I'll change this for you.

Reply:
I am still not sold on calling it addFavorite or combining it in with the addFavorites function. The problem is the names of the old addFavorites and removeFavorites functions. They were not named well. But we cannot change that without changing the api of the toggle favorites library. How do you feel about sendAddFavorite and sendDeleteFavorite?